Snippets/Router/Router_2/API.cs

```diff
@@ -0,0 +1,195 @@
+            outgoingInterface: "Msmq");
+
+        #endregion
+
+        #region lifecycle
+
+        var router = Router.Create(routerConfig);
+
+        await router.Start().ConfigureAwait(false);
+
+        await router.Stop().ConfigureAwait(false);
+
+        #endregion
+    }
+
+    public void ThreeWayRouterConfig()
+    {
+        #region three-way-router
+
+        var routerConfig = new RouterConfiguration("MyRouter");
+
+        routerConfig.AddInterface<SqlTransport>("A", transportExtensions => {});
+        routerConfig.AddInterface<SqlTransport>("B", transportExtensions => {});
+        routerConfig.AddInterface<SqlTransport>("C", transportExtensions => {});
+
+        var staticRouting = routerConfig.UseStaticRoutingProtocol();
+        //Send all messages to endpoints which name starts with Sales via interface A
+        staticRouting.AddRoute(
+            destinationFilter: (iface, destination) => destination.Endpoint.StartsWith("Sales."),
+            destinationFilterDescription: "To Sales", 
+            gateway: null, 
+            iface: "A");
+
+        staticRouting.AddRoute(
+            (iface, destination) => destination.Endpoint.StartsWith("Shipping."), 
+            "To Shipping", null, "B");
+
+        staticRouting.AddRoute(
+            (iface, destination) => destination.Endpoint.StartsWith("Billing."),
+            "To Billing", null, "C");
+
+        #endregion
+    }
+
+    public void SiteBasedRouting()
+    {
+        #region three-way-router
+
+        var routerConfig = new RouterConfiguration("Router.WestEurope");
+
+        routerConfig.AddInterface<MsmqTransport>("Endpoints", transportExtensions => { });
+        routerConfig.AddInterface<AzureStorageQueuesTransport>("Backplane", transportExtensions => { });
+
+        var staticRouting = routerConfig.UseStaticRoutingProtocol();
+
+        //Send all messages from the Backplane interface directly to the destination endpoints
+        staticRouting.AddRoute(
+            (iface, destination) => iface == "Backplane",
+            "From outside", null, "Endpoints");
+
+        //Send all messages to site WestUS through the Backplane interface via Router.WestUS
+        staticRouting.AddRoute(
+            destinationFilter: (iface, destination) => destination.Site == "WestUS",
+            destinationFilterDescription: "To West US",
+            gateway: "Router.WestUS",
+            iface: "Backplane");
+
+        //Send all messages to site EastUS through the Backplane interface via Router.EastUS
+        staticRouting.AddRoute(
+            (iface, destination) => destination.Site == "EastUS",
+            "To East US", "Router.EastUS", "Backplane");
+
+        #endregion
+    }
+
+    public void OtherAPIs()
+    {
+        var routerConfig = new RouterConfiguration("MyRouter");
+
+        #region recoverability
+
+        routerConfig.CircuitBreakerThreshold = 20;
+        routerConfig.DelayedRetries = 10;
+        routerConfig.ImmediateRetries = 10;
```

Comment:
Nothing to do with the doco, but would it not be nicer if you could configure it exactly like [delayed retries](https://docs.particular.net/nservicebus/recoverability/configure-delayed-retries#configuring-delayed-retries)?